instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumMonitorE2ETests.kt

```diff
@@ -624,6 +624,41 @@ class RumMonitorE2ETests {
         }
     }
 
+    /**
+     * apiMethodSignature: com.datadog.android.rum.RumMonitor#fun stopResourceWithError(String, Int?, String, RumErrorSource, String, String?, Map<String, Any?> = emptyMap())
+     */
+    @Test
+    fun rum_rummonitor_stop_resource_with_error_stacktrace() {
+        val testMethodName = "rum_rummonitor_stop_resource_with_error_stacktrace"
+        val viewKey = forge.aViewKey()
+        val viewName = forge.aViewName()
+        val resourceKey = forge.aResourceKey()
+        val statusCode = forge.anInt(min = 400, max = 511)
+        val message = forge.aResourceErrorMessage()
+        val source = forge.aValueFrom(RumErrorSource::class.java)
+        val stackTrace = forge.anAlphaNumericalString()
```

Comment:
you should use `aString()`. There's no guarantee at all that a stack trace will only contain alphanumerical chars. `aString()` will forge a string with any UTF8 character (so smileys, foreign language characters, weird punctuation symbols, …)